docs/how-to/contribute.rst

```diff
@@ -1,3 +1,5 @@
+FIXME pre-commit
+
```

Comment:
Is this stray debug?

Reply:
Yes, that's why I marked the PR as draft. :smiley: